github-orgs/kubeflow/org.yaml

```diff
@@ -821,6 +821,13 @@ orgs:
             privacy: closed
             repos:
               kfctl: write
+          kubeflow-testing:
+            description: Team that will manage testing repo
+            maintainers:
+            - PatrickXYS
```

Comment:
Should we add @Bobgy ? This way @Bobgy will also get notified about relevant issues and can weigh in.

Reply:
Sounds good, added @Bobgy